CMakeLists.txt

```diff
@@ -1373,6 +1373,15 @@ if(ENABLE_CPP)
 endif()
 
 
+# pkg-config support
+include(FindPkgConfig QUIET)
+if(PKG_CONFIG_FOUND)
+  configure_file("${PROJECT_SOURCE_DIR}/tools/pkg-config/stumpless.pc.in" "${PROJECT_BINARY_DIR}/tools/pkg-config/stumpless.pc" @ONLY)
+  install(FILES "${PROJECT_BINARY_DIR}/tools/pkg-config/stumpless.pc"
+    DESTINATION "${CMAKE_INSTALL_PREFIX}/lib/pkgconfig")
+endif()
+
+
```

Comment:
Please move this block under the other installation commands, around line 1000 of the `CMakeLists.txt` file. This way it will be closer to the other install commands, which seems like a more intuitive place for it.

Reply:
Done! 🙂